src/tree_cmd.cpp

```diff
@@ -54,10 +54,16 @@ enum ExtraTreePlacement {
 /** Determines when to consider building more trees. */
 byte _trees_tick_ctr;
 
+static const uint TREE_GROWTH_PROCEDURAL = 7;  ///< Magic value to signal that tree growth is determined procedurally.
```

Comment:
I'm going to need a bit more clarification on this magic number. The comment reads like it's a setting value, but you didn't add to the settings... maybe explain what it's compared with to help me understand?

Reply:
Idk how to explain it any better. There are 3 bits in the map array for growth stage. Values 0-6 were used before and still are. Value 7 was unused, not it means that growth stage is not stored.
By no means should this be configurable.